build/teamcity/cockroach/ci/tests/roachtest_stress_impl.sh

```diff
@@ -24,7 +24,7 @@ log_into_gcloud
 export GCE_PROJECT=${GCE_PROJECT-cockroach-roachstress}
 
 build/teamcity-roachtest-invoke.sh \
-  --cloud=gce \
+  --cloud="${CLOUD-gce}" \
```

Comment:
No need to change since the file uses this throughout and we control the execution environment, but more of a note to myself: `${CLOUD-gce}` and `${CLOUD:-gce}` only differ by how they handle declared but empty variables:

```
> bash foo.sh 
${CLOUD-gce} = gce
${CLOUD:-gce} = gce
> CLOUD= bash foo.sh 
${CLOUD-gce} = 
${CLOUD:-gce} = gce
> CLOUD=foo bash foo.sh 
${CLOUD-gce} = foo
${CLOUD:-gce} = foo